### ISSUE TITLE
Error loading xyzvbin files

### ISSUE THREAD

**Issue by ajtribick** [CelestiaProject/Celestia#1322]
In 1.7.0, if a sampled trajectory is specified using the extension ".xyzv", Celestia will initially look for a file with the extension ".xyzvbin" and try to load that. I'm guessing the reason for this implementation is backwards compatibility with 1.6.x, enabling an add-on creator to supply both .xyzv and .xyzvbin files in a way compatible with 1.6.x and 1.7.

The simplest way to avoid the errors would be to only load the specified extension. This would break the backward-compatibility mechanism. Another simple way would be to check if the file exists before trying to load it, but this still goes against the principle that lookups of different extensions only occur if there is a wildcard character.

If we need to support this backward-compatibility mechanism, an alternative would be to add another key, e.g. `BinarySource` to the `SampledTrajectory` block. E.g. given the existence of files "foo.xyzv" and "foo.xyzvbin":

```
SampledTrajectory { Source "foo.xyzv" BinarySource "foo.xyzvbin" }
```
would load "foo.xyzv" on 1.6.x, and "foo.xyzvbin" on 1.7.0

```
SampledTrajectory { Source "foo.xyzv" BinarySource "does_not_exist.xyzvbin" }
```
would load "foo.xyzv" on 1.6.x and 1.7.0, with an error that the file "does_not_exist.xyzvbin" could not be loaded on 1.7.0.

```
SampledTrajectory { Source "foo.xyzv" }
```
would always load *only* "foo.xyzv" on both 1.6.x and 1.7.0 (would not try "foo.xyzvbin"

```
SampledTrajectory { Source "foo.xyzvbin" }
SampledTrajectory { BinarySource "foo.xyzvbin" }
```
would load "foo.xyzvbin" on 1.7.0, and fail on 1.6.x.

This would obviously be a breaking change for add-ons using .xyzvbin files loaded with the extension ".xyzv".

I tend to lean towards dropping the backward-compatibility aspects, but then again I tend to run a fairly vanilla install. It would be good to get an idea of how common xyzvbin files are in add-ons to figure out how best to proceed here.

**Comment by SevenSpheres** [comment]
I haven't seen xyzvbin files used at all.

**Comment by 375gnu** [comment]
The idea behind such implementation was not a backward compatibility but _caching_ slowly loaded data. That's why they have native byte order instead of specified. It was supposed initially that addons will supply only portable text files and during installation their binary versions will be created (exactly what I did for deb packages of CelestiaContent). Then Celestia Origin team requested to make binary files independent of their text counterparts so they could make their installation files smaller.

So I would just change log level of the error to `verbose` instead of `error` if they are loaded as caches not directly.

**Comment by 375gnu** [comment]
Fixed in 073b28a
